tests/epyccel/modules/arrays.py

```diff
@@ -7,6 +7,22 @@
 a_2d_f = np.array([[1 << j for j in range(21)] for i in range(21)], dtype=int, order='F')
 a_2d_c = np.array([[1 << j for j in range(21)] for i in range(21)], dtype=int)
 
+
+@types('int', 'int')
+@types('int8', 'int8')
+@types('int16', 'int16')
+@types('int32', 'int32')
+@types('int64', 'int64')
+@types('float', 'float')
+@types('float32', 'float32')
+@types('float64', 'float64')
+@types('complex64', 'complex64')
+@types('complex128', 'complex128')
+def array_return_first_element(a, b):
```

Comment:
Can't we handle all these cases using a template? See [here](https://github.com/pyccel/pyccel/blob/master/tutorial/templates.md#templates-using-decorators)

Reply:
Fixed [5421f09](https://github.com/pyccel/pyccel/pull/1166/commits/5421f09d6e770384f5e0a2fad5dc8d9b72b025e5)